### PR TITLE
add units in comment in py/picca/dla.py

### PR DESCRIPTION
**Pull request by echaussidon**
to understand the value of these parameters and the calculation which is done:
Fix in part issue https://github.com/igmhub/picca/issues/584 and issue https://github.com/igmhub/picca/issues/607

### CODE REVIEW

File: py/picca/dla.py [hunk 1]
```diff
@@ -59,4 +61,3 @@ def voigt(a,u):
         y = sp.random.normal(size=nmc)*sp.sqrt(2)
         m = sp.mean(1/(a**2+(y[:,None]-u)**2),axis=0)
         return m*a/sp.sqrt(sp.pi)
-
```
Comment (londumas) [inline py/picca/dla.py]: could you remove this useles change?

File: py/picca/dla.py [hunk 2]
```diff
@@ -17,19 +17,21 @@ def p_voigt_a(la,zabs,nhi):
 
     @staticmethod
     def tau_a(la,zabs,nhi):
-        lam_lya = constants.absorber_IGM["LYA"]
-        gamma = 6.625e8
+        lam_lya = constants.absorber_IGM["LYA"] ##A
```
Comment (londumas) [inline py/picca/dla.py]: space between ## and A

File: py/picca/dla.py [hunk 3]
```diff
@@ -17,19 +17,21 @@ def p_voigt_a(la,zabs,nhi):
...
+        gamma = 6.625e8 ##s^-1
```
Comment (londumas) [inline py/picca/dla.py]: space

File: py/picca/dla.py [hunk 4]
```diff
@@ -17,19 +17,21 @@ def p_voigt_a(la,zabs,nhi):
...
         f = 0.4164
         c = 3e8 ## speed of light m/s
-        b = 30000.
-        nn = 10**nhi
-        lrf = la/(1+zabs)
+        b = 30000. ## = sqrt(2*k*T/m_proton) with T = 5*10^4 ##m.s^-1
+        nn = 10**nhi ## cm^-2
+        lrf = la/(1+zabs) ##A
```
Comment (londumas) [inline py/picca/dla.py]: space

File: py/picca/dla.py [hunk 5]
```diff
@@ -17,19 +17,21 @@ def p_voigt_a(la,zabs,nhi):
...
 
-        u = (c/b)*(lam_lya/lrf-1)
+        u = (c/b)*(lam_lya/lrf-1) ##A
```
Comment (londumas) [inline py/picca/dla.py]: space

**Comment by londumas** [comment]
@echaussidon, Thanks for the work. it looks good, I just requested some small changes though.

**Comment by londumas** [comment]
@echaussidon, thanks for the changes. Is it ready to be merged?

**Comment by echaussidon** [comment]
yes ! :)

**Comment by londumas** [comment]
@echaussidon, thank you very much. Merged.

**Pull request merged**